### ISSUE TITLE
Data USA header search navigates away from map

### ISSUE THREAD

**Issue by zwhitman** [DataUSA/datausa-site#187]
I think the search function in the headerbar is confusing in the map context - expected to act as gazetteer

**Comment by davelandry** [comment]
might be a little difficult to hijack and make interface with the map. maybe we should just hide it on the map page for now?

**Comment by zwhitman** [comment]
def don't mess with the header - this ties into another comment but i think if we move controls out of the header there will be less confusion

**Comment by davelandry** [comment]
@zwhitman with all the changes we've made to the map page, do you think there could still be confusion with the search button?

**Comment by zwhitman** [comment]
yeah - inherently anytime I see a search icon on a map i assume it's a gazetteer. What about if we change the placeholder from "Press Enter to refine search" to something more like "Search DataUSA" on just the map page to help clarify things? Not a great solution but might help.

That said - i still think a map search option would be really helpful. This would prob muddy the water further but I definitely think it'd be a really great feature for the map.

**Comment by davelandry** [comment]
Let's fork the map search conversation: https://github.com/DataUSA/datausa-site/issues/276

I like the idea of changing the site search placeholder text to be more explicit. If we change it, I think it should be a global change. I don't think something like "Search DataUSA" or "Search Profiles" would work on all pages (not just map).
